server/game/player.js

```diff
@@ -1051,12 +1052,29 @@ class Player extends GameObject {
         return this.getEffects(EffectNames.ChangePlayerSkillModifier).reduce((total, value) => total + value, 0);
     }
 
+    honorGained(round = null, phase = null, onlyPositive = false) {
+        return this.honorEvents
+            .filter(event => !round || event.round === round)
+            .filter(event => !phase || event.phase === phase)
+            .filter(event => !onlyPositive || event.amount > 0)
+            .reduce((total, event) => total + event.amount, 0);
+    }
+
     modifyFate(amount) {
         this.fate = Math.max(0, this.fate + amount);
     }
 
     modifyHonor(amount) {
         this.honor = Math.max(0, this.honor + amount);
+        this.honorEvents.push({
+            amount,
+            phase: this.game.currentPhase,
+            round: this.game.roundNumber
+        });
+    }
+
+    resetHonorEvents(round, phase) {
+        this.honorEvents = this.honorEvents.filter(event => event.round !== round && event.phase !== phase);
```

Comment:
Assuming this is a bit of a hack to work with Season of War?  Might want to add a comment explaining that so if there's a later card that is like "restart the conflict phase" people have a chance of realizing this needs to go in there.